updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,134 @@
 [id="updating-disconnected-cluster"]
-= Updating a disconnected cluster to a minor version
+= Updating a disconnected cluster
 include::modules/common-attributes.adoc[]
 :context: updating-disconnected-cluster
 
 toc::[]
 
-You can update, or upgrade, an {product-title} cluster that you installed in a
-restricted network to a minor version by using the web console.
+You can update, or upgrade a disconnected or fully air gapped {product-title} cluster using `oc` commandline.
 
 .Prerequisites
 
+* Have access to a conatiner registry in the disconnected environment to push and pull images. The container registry should be compatible with Docker registry API v2.
 * Have access to the cluster as a user with `admin` privileges.
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
 
-//mirror module
+[IMPORTANT]
+====
+If you are upgrading from earlier release than 4.5.0, you must must use the manual method of creating configmap.
```

Comment:
As far as *creating* the ConfigMap goes, the version that matters is `oc`, not the original cluster version.  Not clear to me what the doc policy is about talking about version numbers, but my impression is that these master docs (so tracking 4.6?) just assume that folks will understand `oc` references to be referenced to (the latest?) 4.6.z `oc`.  If that's true, we can drop this callout.

I did raise "what happens if docs assume latest z-stream `oc` and users are running an earlier `oc` from that z stream that lacks $FEATURE" with @smarterclayton at some point, and he said that while that might happen, it hasn't happened often enough to be a problem.  I think that was in the context of openshift/enhancements#188, but it might have been an out of band discussion, because I don't see anything about it in the PR comments.

Reply:
I agree that the issue is only with the version of `OC` command line. But I am assuming customers use same version of `oc` and OpenShift version. The reason I mentioned this because we plan to back port this document to 4.4 and 4.3. I am not confident what approach will best serve the backporting.